bot/exts/evergreen/bookmark.py

```diff
@@ -37,27 +104,39 @@ async def bookmark(
             await ctx.send(embed=embed)
             return
 
-        embed = discord.Embed(
-            title=title,
-            colour=Colours.soft_green,
-            description=target_message.content
-        )
-        embed.add_field(name="Wanna give it a visit?", value=f"[Visit original message]({target_message.jump_url})")
-        embed.set_author(name=target_message.author, icon_url=target_message.author.avatar_url)
-        embed.set_thumbnail(url=Icons.bookmark)
-
-        try:
-            await ctx.author.send(embed=embed)
-        except discord.Forbidden:
-            error_embed = discord.Embed(
-                title=random.choice(ERROR_REPLIES),
-                description=f"{ctx.author.mention}, please enable your DMs to receive the bookmark",
-                colour=Colours.soft_red
+        def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
```

Comment:
Why are there underscores at the end of these variable names? They don't overwrite anything as far as I can see.

Reply:
copy&paste strikes again D: